src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.System/Pages/SystemCommandPage.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CmdPal.Ext.System.Helpers;
+using Microsoft.CommandPalette.Extensions;
+using Microsoft.CommandPalette.Extensions.Toolkit;
+
+namespace Microsoft.CmdPal.Ext.System.Pages;
+
+public sealed partial class SystemCommandPage : ListPage
+{
+    private SettingsManager _settingsManager;
+
+    public SystemCommandPage(SettingsManager settingsManager)
+    {
+        Title = Resources.Microsoft_plugin_ext_system_page_name;
+        Icon = new IconInfo("\uE72E");
```

Comment:
@niels9001 can we get a better icon for the system commands too? I feel like the `Lock` icon isn't the best option. Maybe something with like, a PC? 

(this is a nit, we can tidy this in post)